### PR TITLE
Update docs for Open/Close API

### PR DESCRIPTION
**Pull request by tlrx**
This is a small update of the Open/Close API docs.

Relates https://github.com/elastic/elasticsearch/pull/43530#pullrequestreview-253374123

### CODE REVIEW

File: docs/reference/indices/open-close.asciidoc [hunk 1]
```diff
@@ -36,6 +59,6 @@ API by setting `cluster.indices.close.enable` to `false`. The default is `true`.
 [float]
 === Wait For Active Shards
 
-Because opening an index allocates its shards, the
+Because opening or closing an index allocates its shards, the
 <<create-index-wait-for-active-shards,`wait_for_active_shards`>> setting on
-index creation applies to the index opening action as well.
+index creation applies to the index `_open` and `_close` actions as well.
```
Comment (ywelsch) [inline docs/reference/indices/open-close.asciidoc]: the `_open` and `_close` index actions

File: docs/reference/indices/open-close.asciidoc [hunk 2]
```diff
@@ -2,19 +2,42 @@
...
+cluster to ensure that enough shard copies are safely kept around
+at all times.
 
 The REST endpoint is `/{index}/_close` and `/{index}/_open`. For
 example:
 
 [source,js]
 --------------------------------------------------
 POST /my_index/_close
+{
```
Comment (ywelsch) [inline docs/reference/indices/open-close.asciidoc]: this will look like this is the body of the request, whereas you would like to show this as response.

**Review by ywelsch — CHANGES_REQUESTED** [review]

File: docs/reference/indices/open-close.asciidoc (hunk 2 above)
Comment (tlrx) [reply docs/reference/indices/open-close.asciidoc]: Looks like I didn't write some docs for a long time... Thanks

File: docs/reference/indices/open-close.asciidoc [hunk 3]
```diff
@@ -2,22 +2,68 @@
...
 
+[source,js]
+--------------------------------------------------
 POST /my_index/_open
 --------------------------------------------------
 // CONSOLE
-// TEST[s/^/PUT my_index\n/]
+// TEST[s/^/PUT my_index\nPOST my_index\/_close\n/]
+
+Which will yield the following response:
```
Comment (ywelsch) [inline docs/reference/indices/open-close.asciidoc]: s/Which/which/

File: docs/reference/indices/open-close.asciidoc [hunk 4]
```diff
@@ -2,22 +2,68 @@
...
+The following example shows how to close an index:
 
 [source,js]
 --------------------------------------------------
 POST /my_index/_close
+--------------------------------------------------
+// CONSOLE
+// TEST[s/^/PUT my_index\n/]
+
+And this will return the following response:
```
Comment (ywelsch) [inline docs/reference/indices/open-close.asciidoc]: This will return the following response:

**Review by ywelsch — APPROVED** [review]

**Comment by tlrx** [comment]
Thanks @ywelsch

**Pull request merged**